README.md

```diff
@@ -9,13 +9,16 @@ in pure go/cgo, rather than cgo bindings for libsigar.
 
 ## Test drive
 
-    $ go get github.com/cloudfoundry/gosigar
-    $ cd $GOPATH/src/github.com/cloudfoundry/gosigar/examples
+    $ git clone https://github.com/cloudfoundry/gosigar.git
+    $ cd gosigar/examples
     $ go run uptime.go
+    $ go run df.go
+    $ go run free.go
+    $ go run ps.go
 
 ## Supported platforms
 
-Currently targeting modern flavors of darwin and linux.
+Currently targeting modern flavors of macOS (darwin) and Linux.
```

Comment:
If we're going to properly capitalize "Linux", "Darwin" should get the same treatment.

Reply:
Done.

What about Windows, should that be added?

Reply:
Good catch. That should be added.

Reply:
Done in f79a8774347bcfc6e9768e2c1afc44f463a860b6